CLDConfig/CLDReconstruction.py

```diff
@@ -0,0 +1,1155 @@
+    from Configurables import LcioEvent
+    read = LcioEvent()
+    read.OutputLevel = WARNING
+    read.Files = input_files
+    algList.append(read)
+elif CONFIG["InputMode"] == "EDM4hep":
+    evtsvc.inputs = input_files
+    from Configurables import PodioInput
+    inp = PodioInput('InputReader')
+    inp.collections = [
+      'EventHeader',
+      'MCParticles',
+      'VertexBarrelCollection',
+      'VertexEndcapCollection',
+      'InnerTrackerBarrelCollection',
+      'OuterTrackerBarrelCollection',
+      'ECalEndcapCollection',
+      'ECalEndcapCollectionContributions',
+      'ECalBarrelCollection',
+      'ECalBarrelCollectionContributions',
+      'HCalBarrelCollection',
+      'HCalBarrelCollectionContributions',
+      'InnerTrackerEndcapCollection',
+      'OuterTrackerEndcapCollection',
+      'HCalEndcapCollection',
+      'HCalEndcapCollectionContributions',
+      'HCalRingCollection',
+      'HCalRingCollectionContributions',
+      'YokeBarrelCollection',
+      'YokeBarrelCollectionContributions',
+      'YokeEndcapCollection',
+      'YokeEndcapCollectionContributions',
+      'LumiCalCollection',
+      'LumiCalCollectionContributions',
+    ]
+    inp.OutputLevel = WARNING
+
+
+MyAIDAProcessor = MarlinProcessorWrapper("MyAIDAProcessor")
+MyAIDAProcessor.OutputLevel = WARNING
+MyAIDAProcessor.ProcessorType = "AIDAProcessor"
+MyAIDAProcessor.Parameters = {
+                              "Compress": ["1"],
+                              "FileName": [f"{output_basename}_aida"],
+                              "FileType": ["root"]
+                              }
+
+if CONFIG["InputMode"] == "LCIO":
+    from Configurables import EDM4hep2LcioTool
+    EDM4hep2Lcio = EDM4hep2LcioTool("EDM4hep2Lcio")
+    EDM4hep2Lcio.convertAll = False
+    EDM4hep2Lcio.collNameMapping = {
+        'MCParticles':                     'MCParticle',
+        'VertexBarrelCollection':          'VertexBarrelCollection',
+        'VertexEndcapCollection':          'VertexEndcapCollection',
+        'InnerTrackerBarrelCollection':    'InnerTrackerBarrelCollection',
+        'OuterTrackerBarrelCollection':    'OuterTrackerBarrelCollection',
+        'InnerTrackerEndcapCollection':    'InnerTrackerEndcapCollection',
+        'OuterTrackerEndcapCollection':    'OuterTrackerEndcapCollection',
+        'ECalEndcapCollection':            'ECalEndcapCollection',
+        'ECalBarrelCollection':            'ECalBarrelCollection',
+        'HCalBarrelCollection':            'HCalBarrelCollection',
+        'HCalEndcapCollection':            'HCalEndcapCollection',
+        'HCalRingCollection':              'HCalRingCollection',
+        'YokeBarrelCollection':            'YokeBarrelCollection',
+        'YokeEndcapCollection':            'YokeEndcapCollection',
+        'LumiCalCollection':               'LumiCalCollection',
+    }
+    EDM4hep2Lcio.OutputLevel = DEBUG
+
+InitDD4hep = MarlinProcessorWrapper("InitDD4hep")
```

Comment:
Use the GeoSvc now: https://github.com/key4hep/k4MarlinWrapper/tree/main/doc/starterkit/k4MarlinWrapperCLIC#dd4hep-geometry-information